### PR TITLE
Export files refactor

### PR DESCRIPTION
**Pull request by chr1sgrant**

### CODE REVIEW

**Comment by chr1sgrant** [comment]
depends on antha-lang/antha #302

File: an/Data/DNA/GeneDesign/GeneDesign.an [hunk 1]
```diff
@@ -52,7 +52,11 @@ Steps {
 	}
 	
 	// Look up the restriction enzyme
-	EnzymeInf, _ := lookup.TypeIIsLookup(RE)
+	EnzymeInf, err := lookup.TypeIIsLookup(RE)
```
Comment (jarpinosynthace) [inline an/Data/DNA/GeneDesign/GeneDesign.an]: This variable RE needs to change to REName or RestrictionEnzymeName or something like that RE is too ambiguous.

File: an/Data/DNA/GeneDesign/GeneDesign_seq.an [hunk 2]
```diff
@@ -54,7 +54,10 @@ Steps {
 	VectorSeq := wtype.MakePlasmidDNASequence("Vector",Vector)
 
 	// Look up the restriction enzyme
-	EnzymeInf, _ := lookup.TypeIIsLookup(RE)
+	EnzymeInf, err := lookup.TypeIIsLookup(RE)
```
Comment (jarpinosynthace) [inline an/Data/DNA/GeneDesign/GeneDesign_seq.an]: Same as previous rename variable RE

**Comment by chr1sgrant** [comment]
I've made those changes to variable names but since that was legacy and a separate issue to this PR good practice is to handle that issue in a separate PR

**Pull request merged**